Library/Homebrew/cleanup.rb

```diff
@@ -321,7 +321,7 @@ def cleanup_path(path)
         yield
       end
 
-      @disk_cleanup_size += disk_usage
+      @disk_cleanup_size += disk_usage - path.disk_usage
```

Comment:
The correction should only be applied when actually removing the files, not when `dry_run?` is `true`.

Reply:
Should it really? Would be weird if you ran a dry run and it says 1.5GB will be freed, and then you run it and it doesn't free 1.5GB. So maybe we should show both values, minimum and maximum that can be freed?

Reply:
No wait, you're totally right, that wouldn't work at all :)